src/contributors page/contributor.html

```diff
@@ -14,8 +14,7 @@
   <!-- Latest compiled and minified JavaScript -->
   <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
 
-  <link rel="stylesheet" href="contributor.css">
-
+  <link rel="stylesheet" href=".././assets/contributor.css">
```

Comment:
only ../assets/contributor.css, should do the work I guess

incase of any error, let me know!

Reply:
try this if it works update the PR